lib/tasks/invite_code.rake

```diff
@@ -1,16 +1,22 @@
-
+require 'base64'
+require 'securerandom'
```

Comment:
is this needed?
I think if the InviteCode.new needs it it should go into the invite code model. And it is there already.